docs/src/main/paradox/routing-dsl/directives/form-field-directives/formFields.md

```diff
@@ -31,14 +31,14 @@ as required, optional, or repeated, or to filter requests where a form field has
 `"color"`
 : extract value of field "color" as `String`
 
-`"color".?`
-: extract optional value of field "color" as `Option[String]`
+`"color".optional`
```

Comment:
I'm a bit conflicted about deemphasizing the symbolic operators because they seem quite useful in this restricted setting where they are much more concise than the textual counterparts.

On the other hand, after reading the changes in the examples below, maybe it's just fine to use the names in most cases. The operators are still there if people would like to use them.